airbyte-integrations/connectors/source-square/source_square/square.yaml

```diff
@@ -0,0 +1,292 @@
+      $ref: "*ref(definitions.base_stream_page_json_limit.retriever)"
+      paginator:
+        pagination_strategy:
+          page_size: 200
+        page_size_option:
+          inject_into: "body_json"
+          field_name: "limit"
+
+  team_members_stream:
+    $ref: "*ref(definitions.base_stream_page_json_limit)"
+    $options:
+      name: "team_members"
+      primary_key: "id"
+      path: "/team-members/search"
+
+  team_member_wages_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "team_member_wages"
+      primary_key: "id"
+      path: "/labor/team-member-wages"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      requester:
+        $ref: "*ref(definitions.requester)"
+        request_options_provider:
+          request_parameters:
+            sort_order: ASC
+            sort_field: CREATED_AT
+      paginator:
+        pagination_strategy:
+          page_size: 200
+  locations_stream:
+    $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "locations"
+      primary_key: "id"
+      path: "/locations"
+
+  categories_stream:
+    $ref: "*ref(definitions.base_catalog_objects_stream)"
+    $options:
+      name: "categories"
+      object_type: "CATEGORY"
+      path: "/catalog/search"
+
+  items_stream:
+    $ref: "*ref(definitions.base_catalog_objects_stream)"
+    $options:
+      name: "items"
+      object_type: "ITEM"
+      path: "/catalog/search"
+
+  discounts_stream:
+    $ref: "*ref(definitions.base_catalog_objects_stream)"
+    $options:
+      name: "discounts"
+      object_type: "DISCOUNT"
+      path: "/catalog/search"
+
+  taxes_stream:
+    $ref: "*ref(definitions.base_catalog_objects_stream)"
+    $options:
+      name: "taxes"
+      object_type: "TAX"
+      path: "/catalog/search"
+
+  modifier_list_stream:
+    $ref: "*ref(definitions.base_catalog_objects_stream)"
+    $options:
+      name: "modifier_list"
+      object_type: "MODIFIER_LIST"
+      path: "/catalog/search"
+
+  refunds_stream:
+    $ref: "*ref(definitions.base_incremental_stream)"
+    $options:
+      name: "refunds"
+      primary_key: "id"
+      path: "/refunds"
+    retriever:
+      $ref: "*ref(definitions.base_incremental_stream.retriever)"
+
+  payments_stream:
+    $ref: "*ref(definitions.base_incremental_stream)"
+    $options:
+      name: "payments"
+      primary_key: "id"
+      path: "/payments"
+    retriever:
+      $ref: "*ref(definitions.base_incremental_stream.retriever)"
+
+  orders_stream:
+    $ref: "*ref(definitions.base_stream_page_json_limit)"
+    $options:
+      name: "orders"
+      primary_key: "id"
+      path: "/orders/search"
+    retriever:
+      $ref: "*ref(definitions.base_stream_page_json_limit.retriever)"
+      requester:
+        $ref: "*ref(definitions.base_stream_page_json_limit.retriever.requester)"
+        http_method: "POST"
+        request_options_provider:
+          request_body_json:
+            limit: "{{ 500 }}"
```

Comment:
why is this between `{{}}` ? shouldn't `limit: 500` suffice?

Reply:
request_options_provider accepts only interpolated strings